styles.css

```diff
@@ -1,3 +1,125 @@
-h1 {
-  color: red;
+
+/* styles applying to all */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+svg {
+  fill: #fff;
+}
+
+.body {
+  height: 100vh;
+  width: 100%;
+}
+
+/* styles for top bar */
+
+.top {
+  background-color: #3c3a39;
+  height: 10%;
+  width: 100%;
+  color: #fff;
+}
+
+.navigation_bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50%;
+  width: 100%;
+}
+
+.navigation_bar a {
+  margin: 1em 1em 0 1em;
+  font-family: 'Roboto', sans-serif;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navigation_bar svg {
+  margin: 1em 1em 0 1em;
+}
+
+.navigation_bar svg:hover {
+  cursor: pointer;
+}
+
+/* main section styles */
+
+.content {
+  background-color: #1c1a19;
+  background-image: url(img/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  height: 90%;
+}
+
+.content .details {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  width: 90%;
+  height: 80%;
+  margin: 0 1.5em 0 1.5em;
+}
+
+.content h1 {
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-size: 3em;
+  font-weight: 400;
+}
```

Comment:
- [OPTIONAL] KIndly make sure the main title has the typography styles as the provided Figma design (`font-weight`).

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/227333818-2b6831ab-41d4-4b4f-be37-764f1522d322.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/227333911-4c850414-68df-4066-a2e2-45c02510431f.png)

Reply:
Alright, Thank you!